contracts/HashRegistrarSimplified.sol

```diff
@@ -253,12 +288,12 @@ contract Registrar {
     function startAuction(bytes32 _hash) registryOpen() {
         var mode = state(_hash);
         if(mode == Mode.Auction) return;
-        if(mode != Mode.Open) throw;
-
+        if(mode != Mode.Open || !isAllowed(_hash, now)) throw;
```

Comment:
Won't `mode == Mode.Unavailable` mean that the first part of this if statement fails?